content/flux/v0.x/query-data/influxdb.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Query InfluxDB
+list_title: InfluxDB
+description: >
+  Use [`from()`](/flux/v0.x/stdlib/universe/from/) and [`range`](/flux/v0.x/stdlib/universe/range/)
+  to query data from InfluxDB using Flux.
+menu:
+  flux_0_x:
+    name: InfluxDB
+    parent: Query data sources
+weight: 101
+related:
+  - /influxdb/cloud/query-data/optimize-queries/, Optimize Flux queries for InfluxDB
+list_code_example: |
+  ```js
+  from(bucket: "example-bucket")
+    |> range(start: -1h)
+  ```
+---
+
+Use [`from()`](/flux/v0.x/stdlib/universe/from/) and [`range`](/flux/v0.x/stdlib/universe/range/)
+to query data from InfluxDB using Flux.
+Provide the **bucket name** or **bucket ID** to query.
+
+{{% note %}}
+InfluxDB requires queries to be time-bound, so `from()` must always be followed by
+[`range()`](/flux/v0.x/stdlib/universe/range/).
+{{% /note %}}
+
+```js
+from(bucket: "example-bucket")
+  |> range(start: -1h)
+```
+
+### Query InfluxDB Cloud or 2.x remotely
+To query InfluxDB Cloud or 2.x remotely, provide the following parameters
+in addition to **bucket** or **bucketID**.
+
+- **host**: [InfluxDB Cloud region URL](/influxdb/cloud/reference/urls/) or
+  [InfluxDB URL](/{{< latest "influxdb" >}}/reference/urls/)
+- **org** or **orgID**: InfluxDB organization name or ID
+- **token**: InfluxDB [authentication token](/influxdb/cloud/security/tokens/)
+
+```js
+from(
+  bucket: "example-bucket",
+  host: "http://localhost:8086",
+  org: "example-org",
+  token: "mYSup3r5Ecr3T70keN"
+)
+```
+
+### Query InfluxDB 1.x
+To query InfluxDB 1.x, use the `database-name/retention-policy-name` naming
+convention for your bucket name.
+For example, to query data from the `autogen` retention policy in the `telegraf` database:
+
+```js
+from(bucket: "telegraf/autogen")
+  |> range(start: -30m)
+```
+
+To query the **default retention policy** in a database, use the same bucket naming
```

Comment:
Shall we provide context for default retention policy? 
/influxdb/v1.8/query_language/explore-schema/#run-a-show-retention-policies-query-with-the-on-clause
or here?: 
/influxdb/v1.8/query_language/manage-database/#create-a-retention-policy